apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex

```diff
@@ -18,10 +18,12 @@
 
 defmodule Astarte.Pairing.APIWeb.Router do
   use Astarte.Pairing.APIWeb, :router
+  alias Astarte.Pairing.APIWeb.Plug.LogRealm
 
   pipeline :realm_api do
     plug :accepts, ["json"]
-    plug Astarte.Pairing.APIWeb.Plug.LogRealm
+    plug LogRealm
```

Comment:
Nit: this line is inconsistent with the rest of the code. Either all plugs should be aliased, or none